stable/rethinkdb/templates/rethinkdb-secrets.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  rethinkdb-password: {{ .Values.rethinkdbPassword | b64enc | quote }}
```

Comment:
@prydonius I agree there should be a secure default. But the issue people can run into is if they do a `helm upgrade` in the future, it will generate a new password unless they put the first password that was auto-generated in their local `values.yaml` file they are using to configure the k8s deployment.  But if that's an acceptable risk, then I'm all for auto-generating :).